packages/assets/express/frontend.js

```diff
@@ -33,6 +41,7 @@ module.exports = (server) => {
       console.error('frontend fetch failed', result.url, result.status)
       return res.status(result.status).end()
     }
+    result.headers.set('Link', `<${frontendUrl}>; rel="canonical"`)
```

Comment:
Result is already done at this point, so this seems wired.
```
return returnImage({
      response: res,
      stream: result.body,
      headers: new Headers({
        ...headers.get().split(','),
        Link: `<${frontendUrl}>; rel="canonical"`
      }),
      options: {
        ...req.query,
        webp: !!webp
      }
    })
```
Seems to complicated as well. And I'm not sure how to get Headers in an isomorphic way. Can I suggest you just add a comment, that you add it there, so that returnImage copies is to the "real" response?